src/makeRobotAccountant.js

```diff
@@ -6,7 +6,30 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = (param) => {
+    let result = 0;
+    let msg;
+
+    count++;
+
+    result += param;
+
+    const innerGetSum = (anotherNum) => {
+      result += anotherNum;
+
+      count === 2 || count % 2 !== 0
+        ? msg = result
+        : msg = `Bzzz... Error!`;
+
+      return msg;
```

Comment:
```suggestion
      return count === 2 || count % 2 !== 0
        ? param + anotherNum
        : `Bzzz... Error!`;
```
you can make it like this, so you will need only 1 variable

Reply:
good point! fixed.